fairer-iOS/fairer-iOS/Global/UIComponent/MainButton.swift

```diff
@@ -5,4 +5,62 @@
 //  Created by Mingwan Choi on 2022/09/10.
 //
 
-import Foundation
+import UIKit
+import SnapKit
```

Comment:
p5;

코딩 컨벤션 중 하나인데, first party들은 위에 쓰고 third party들은 한칸띄고 아래에 쓰는 건 어떨까요 ??
각 라이브러리들은 알파벳순으로 정렬하구요 ! ^______^
```suggestion
import UIKit

import SnapKit
```